granulate_utils/linux/elf.py

```diff
@@ -183,3 +183,10 @@ def get_libc_type(elf: ELFType) -> LibcType:
             return LibcType.STATIC_LIBC
 
         return LibcType.STATIC_NO_LIBC
+
+
+def elf_is_stripped(elf: ELFType) -> bool:
```

Comment:
This makes sense, I assume you tested it per my suggestions [here](https://github.com/Granulate/granulate-utils/pull/160#pullrequestreview-1494813829)?

Reply:
Yes I tested it